spec/inputs/tcp_spec.rb

```diff
@@ -241,11 +241,10 @@
     events = input(conf) do |pipeline, queue|
       event_count.times do |i|
         socket = Stud::try(5.times) { TCPSocket.new("127.0.0.1", port) }
-        socket.puts("#{i}")
+        socket.syswrite("#{i}\n")
```

Comment:
@jordansissel we do, test fails without it, we don't have a blocking read here and the timing could be unfortunate enough to make this fail without a flush call.
(fails in 2/3 of all runs if I remove the flush here on my Linux box)